### PR TITLE
re-organize sections

### PR DESCRIPTION
**Pull request by tausani-ah-chong**
# Describe your changes

1. Swapped `Contact` with `Contributors`
2. Updated nav bar text from "Reach out" to "Contact Us"
3. Added `About` to nav bar as it was missing
4. Added `Section` key value pair, to be more explicit and less bug prone if we were to rearrange again

## Issue ticket number and link

https://github.com/Samoa-Stack-Overflow/sso-website/issues/33

## Checklist before requesting a review

- [x] I have performed a self-review of my code
- [ ] If it is a core feature, I have added thorough tests.
- [ ] Will this be part of a product update? If yes, please write one phrase about this update.

### CODE REVIEW

**Review by westerandr — CHANGES_REQUESTED** [review]
Malo uso! Looks good 🙂 

Just one small addition please, are you able to swap the contributors and reach out links on the nav bar 🙏

**Comment by tausani-ah-chong** [comment]
@westerandr Seki I can make that change!

A couple things I took the liberty to do extra while making the change:

1. Can I suggest renaming "Reach out" to "Contact Us". I feel is more consistent with code + the copy in the UI
2. I noticed the 'About' section was missing in the nav bar, so Ive added in. But I also don't have context, and there might be a reason why it wasn't there. Happy to remove if intentionally not there.
3. I created a `SectionName` key value pair, as I noticed when I swapped the `Contributors` & `Contact` in `menuItems` the refs were named `section1`, `section2`, `section3`... and thought having `SectionName` is more explicit and bug prone

Thanks for reviewing uso!

File: src/lib/utils/menuItems.js [hunk 1]
```diff
@@ -1,24 +1,37 @@
 import { IconCar } from '@tabler/icons-svelte';
 
+export const SectionName = {
+  HERO: 'Hero',
+  MEDIA: 'Media',
+  CONTRIBUTORS: 'Contributors',
+  CONTACT: 'Contact',
+  ABOUT: 'About',
+}
```
Comment (tausani-ah-chong) [inline src/lib/utils/menuItems.js:9]: Refer to point 3

Explicit naming

File: src/lib/utils/menuItems.js [hunk 2]
```diff
@@ -1,24 +1,37 @@
...
   {
-    ref: '#section4',
+    ref: `#${SectionName.CONTRIBUTORS}`,
     name: 'Contributors',
     icon: IconCar
-  }
+  },
+  {
+    ref: `#${SectionName.CONTACT}`,
+    name: 'Contact Us',
```
Comment (tausani-ah-chong) [inline src/lib/utils/menuItems.js:34]: Refer to point 1.

Renamed to "Contact Us" for consistency

File: src/lib/utils/menuItems.js [hunk 3]
```diff
@@ -1,24 +1,37 @@
...
-    ref: '#section1',
+    ref: `#${SectionName.HERO}`,
     name: 'Welcome',
     icon: IconCar
   },
   {
-    ref: '#section2',
-    name: 'Media',
+    ref: `#${SectionName.ABOUT}`,
+    name: 'About',
```
Comment (tausani-ah-chong) [inline src/lib/utils/menuItems.js:19]: Refer to point 2.

Was missing in nav bar

File: src/lib/utils/menuItems.js [hunk 4]
```diff
@@ -1,24 +1,37 @@
...
     icon: IconCar
   },
   {
-    ref: '#section3',
-    name: 'Reach Out',
+    ref: `#${SectionName.MEDIA}`,
+    name: 'Media',
     icon: IconCar
```
Comment (tausani-ah-chong) [inline src/lib/utils/menuItems.js:25]: I also noticed that every item as the same icon `IconCar`

Ive [created an issue](https://github.com/Samoa-Stack-Overflow/sso-website/issues/49) for this to be addressed if need be. 

For now I won't bloat the PR even more than it needs to be

**Review by westerandr — APPROVED** [review]

**Comment by westerandr** [comment]
LGTM uso! Thanks for contributing 🤝

**Pull request merged**